framework/include/outputs/Output.h

```diff
@@ -215,6 +215,12 @@ class Output : public MooseObject,
   /// End outputting time
   Real _end_time;
 
+  /// Start outputting at this time step
+  int _start_step;
```

Comment:
@bwspenc, @YaqiWang, This isn't @jessecarterMOOSE's issue, but do we need to support having a negative time_step number? I'm looking through MOOSE and I can't see any place where we set it negative but it's currently declared as an int. In fact, we have a few places where we have to cast it to unsigned to compare to all of the unsigned values that we use everywhere else in MOOSE. It would be nice to know if we really need this and start working on getting it out of here if we don't. 

@jessecarterMOOSE - No action on your part.

Reply:
I do not need for now. Is this there because someone is thinking to march in time backward? We never did adjoint calculations in time, where possibly it is clearer to set dt to negative numbers. We possibly can still just increment time step from 0.

Reply:
Good to know, I'll open a ticket to change step over to unsigned.